crates/ursa-rpc-service/src/client/mod.rs

```diff
@@ -1,4 +1,121 @@
-pub mod functions;
 mod rpc;
 
+use crate::api::{
+    NetworkGetFileParams, NetworkGetParams, NetworkGetResult, NetworkPutFileParams,
+    NetworkPutFileResult, NETWORK_GET, NETWORK_GET_FILE, NETWORK_PUT_FILE,
+};
+use jsonrpc_v2::{Error, RequestObject};
 pub use rpc::*;
+use serde::de::DeserializeOwned;
+use std::net::Ipv4Addr;
+use tracing::{debug, error, info};
+
+pub type Result<T> = anyhow::Result<T, Error>;
+
+pub struct Client {
+    server_addr: Ipv4Addr,
+    server_port: u16,
+}
+
+impl Default for Client {
+    fn default() -> Self {
+        Client::new(Ipv4Addr::from([0, 0, 0, 0]), 4069)
```

Comment:
can we use a `const DEFAULT_PORT` to have this in once place consistent with the ServerConfig default